starter-code/src/functions-and-arrays.js

```diff
@@ -1,93 +1,454 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers(x, y) {
+  return Math.max(x, y);
+}
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
+];
+
+function findLongestWord(arr) {
+  let longest = "";
+  if (arr.length === 0) {
+    return null;
+  } else {
```

Comment:
no need to have the else here because you have a return in the if. 

the rest will only get evaluated if the if is not evaluating to true